build.gradle

```diff
@@ -39,7 +39,7 @@ buildscript {
         appcompatVersion = "1.1.0"
         ktxCoreVersion = "1.2.0"
         securityVersion = "1.0.0-rc02"
-        materialVersion = "1.2.0-alpha06"
+        materialVersion = "1.3.0-alpha01"
```

Comment:
@ioane-bendingspoons so this is needed for an icon tinting issue right? Could you link a reference please.

Reply:
From official changelog:
> Fixed merge icon and view state when setting icon color. ([33700b0](https://github.com/material-components/material-components-android/commit/33700b0cd35a41914dccf2f654d750209bc4c254))

Reply:
I'm not sure if the bug was 100% fixed, since I still needed to add this line

```Kotlin
        searchInput.setOnFocusChangeListener { _, hasFocus ->
            searchInputLayout.isActivated = hasFocus
            searchInputLayout.refreshStartIconDrawableState()
        }
```
Which activates icon when searchInput is focused. But `refreshStartIconDrawableState` method, has been added in new lib-version.